src/main/java/org/tailormap/api/security/OIDCRepository.java

```diff
@@ -84,6 +90,13 @@ public OIDCRegistrationMetadata getMetadataForRegistrationId(String id) {
     } else {
       metadata.showForViewer = true;
     }
+    OIDCConfiguration config =
+        oidcConfigurationRepository.findById(Long.valueOf(id)).orElse(null);
```

Comment:
@steinkobben @matthijsln  This will fail when the id is a string, such as "static" on line 88 above

see https://b3partners.atlassian.net/browse/HTM-1686